CHANGES

```diff
@@ -4,8 +4,10 @@ Pint Changelog
 0.25 (unreleased)
 -----------------
 
+- Fix the default behaviour for pint-convert (cli) for importing uncertainties package
 - Added mu and mc as alternatives for SI micro prefix
 - Added ℓ as alternative for liter
+- Fix the default behaviour for pint-convert (cli) for importing uncertainties package (PR #2032, Issue #2016)
```

Comment:
Looks like a duplicate.

Reply:
Thanks @doronbehar !

I think, I did it during rebase. Related PR https://github.com/hgrecco/pint/pull/2034